scripts/run_e2e_tests.py

```diff
@@ -117,6 +117,7 @@
     'additionaleditorfeaturesmodals': RERUN_POLICY_ALWAYS,
     'additionalplayerfeatures': RERUN_POLICY_ALWAYS,
     'adminpage': RERUN_POLICY_NEVER,
+    'blogdashboard': RERUN_POLICY_KNOWN_FLAKES,
```

Comment:
@U8NWXD is it expected to add new suite with known flake policy?

Reply:
No. New suites should be added with `RERUN_POLICY_NEVER` (since they shouldn't flake)

Reply:
Done